### PR TITLE
Show QR code as image in terminal

### PR DESCRIPTION
**Pull request by kevva**

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -28,4 +31,18 @@ const cli = meow([`
...
 });
 
-m(cli.flags).then(res => console.log(res));
+m(cli.flags).then(res => {
+	if (cli.flags.image) {
+		const buf = res.replace('data:image/png;base64,', '');
+
+		termImg(Buffer.from(buf, 'base64'), {
+			fallback: () => {
+				console.error('iTerm >=2.9 required to show QR code');
```
Comment (sindresorhus) [inline cli.js]: You also use this as a fallback: https://github.com/gtanner/qrcode-terminal

Comment (kevva) [reply cli.js]: Good idea. Had to create a new method (`.generateString`) for it.

**Comment by kevva** [comment]
Maybe you want to extract the CLI into an own module since we've introduced two more dependencies in addition to `meow` for it now.

**Comment by gillstrom** [comment]
Grejt kevvsa, cool!

I will need to extract it indeed.

**Comment by kevva** [comment]
Cool!

**Pull request merged**